runing/schemas.py

```
@@ -0,0 +1,22 @@
+from pydantic import BaseModel 
+from typing import Optional
+
+class UserBase(BaseModel):
+    email: str 
+    name: str
+    is_active: bool = True
+
+class UserCreate(UserBase):
+    pass
+
+class UserUpdate(baseModel):
```

Ruff (0.11.9)

Undefined name `baseModel`

(F821)